### PR TITLE
Feature/named entities refs

### PR DESCRIPTION
**Pull request by ChiaraDipi**
Add support (parser and component) for references to named entities in the text

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -97,6 +98,7 @@ export function initializeApp(appConfig: AppConfig) {
     VersionPanelComponent,
     WitnessPanelComponent,
     XmlBeautifyPipe,
+    NamedEntityRefComponent,
```
Comment (szenzaro) [inline src/app/app.module.ts]: Reorder alphabetically

File: src/app/components/named-entity-ref/named-entity-ref.component.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
+@import "../../../assets/scss/themes";
+@import "../../../assets/scss/mixins";
+@function get-color($key) {
+  @return map-fetch($editionColors, namedEntities $key);
+}
+
+@mixin namedEntityRefColors($color-base, $color-middle, $color-darker) {
+  /* &.location-mainText, &.location-pinned {*/
```
Comment (szenzaro) [inline src/app/components/named-entity-ref/named-entity-ref.component.scss]: Remove comments

File: src/app/components/named-entity-ref/named-entity-ref.component.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  selector: 'evt-named-entity-ref',
+  templateUrl: './named-entity-ref.component.html',
+  styleUrls: ['./named-entity-ref.component.scss'],
+})
+@register
+export class NamedEntityRefComponent {
+  @Input() data: NamedEntityRefData;
+
+  entity$ = this.neParserService.namedEntities$.pipe(
+    share(),
```
Comment (szenzaro) [inline src/app/components/named-entity-ref/named-entity-ref.component.ts]: ??

File: src/app/components/named-entity-ref/named-entity-ref.component.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    map(ne => ne.all.entities.find(e => e.id === this.data.entityId) || 'notFound'),
```
Comment (szenzaro) [inline src/app/components/named-entity-ref/named-entity-ref.component.ts]: `'notFound'` is not af `NamedEntity` type!

File: src/app/components/named-entity-ref/named-entity-ref.component.ts (hunk 3 above)
Comment (ChiaraDipi) [reply src/app/components/named-entity-ref/named-entity-ref.component.ts]: I really don't know why I used it. It's probably a copy-paste from somewhere else. I'll remove it!

File: src/app/components/named-entity-ref/named-entity-ref.component.ts (hunk 4 above)
Comment (ChiaraDipi) [reply src/app/components/named-entity-ref/named-entity-ref.component.ts]: I need a way to distinguish among three different statuses: 
- XML Not yet parsed => `undefined`
- XML Parsed and entity found => `NamedEntity` object
- XML Parsed but entity not found => `'notFound'` string

Do you have a better implementation to suggest?

Comment (szenzaro) [reply src/app/components/named-entity-ref/named-entity-ref.component.ts]: You need to change the entities parser, let's leave it as it is but it should be changed

**Pull request merged**